README.aspnet.md

```diff
@@ -87,6 +87,22 @@ Tag | Dockerfile
 
 You can retrieve a list of all available tags for dotnet/framework/aspnet at https://mcr.microsoft.com/v2/dotnet/framework/aspnet/tags/list.
 
+# Version Compatibility
+
+Version Tag | OS Version | Supported .NET Versions
+-- | -- | --
+`4.8` | _all supported_ | 4.8
```

Comment:
The rendered code formatting within a table on DockerHub isn't a viable option IMO.  The dark alternating row colors renders it useless.

![image](https://user-images.githubusercontent.com/8290530/98987506-e1d9e480-24eb-11eb-961b-76eda54e994c.png)

Reply:
Fixed